src/renderer/marketplace/notarization/common/toc-disagreement.jsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { push } from 'connected-react-router';
+import { Typography, Button, withStyles } from '@material-ui/core';
+import { CloseButtonIcon } from 'selfkey-ui';
+import MarketplaceNotariesComponent from './marketplace-notaries-component';
+
+const styles = theme => ({
+	container: {
+		margin: '0 auto',
+		maxWidth: '500px',
+		position: 'relative',
+		width: '100%'
+	},
+	containerHeader: {
+		alignItems: 'flex-start',
+		background: '#2A3540',
+		display: 'flex',
+		justifyContent: 'flex-start',
+		padding: '20px 30px',
+		'& div': {
+			display: 'inline-block',
+			color: '#FFF'
+		}
+	},
+	closeIcon: {
+		position: 'absolute',
+		right: '-20px',
+		top: '-20px'
+	},
+	contentContainer: {
+		border: '1px solid #303C49',
+		borderRadius: '4px',
+		boxShadow: '0 50px 70px -50px black',
+		padding: '30px'
+	},
+	text: {
+		marginBottom: '40px'
+	},
+	buttons: {
+		marginBottom: '20px'
+	},
+	requestBtn: {
+		marginRight: '20px'
+	}
+});
+
+class NotarizationTOCdisagreementComponent extends MarketplaceNotariesComponent {
+	state = {};
+
+	onBackClick = () => this.props.dispatch(push(this.processPath()));
+	onReturnClick = () => this.props.dispatch(push(this.tocPath()));
```

Comment:
I would remove the state and click handlers lift them up to the container component.
This will allow easy testing of this component and display on storybook (same issue we discussed previously), example, instead of onClick={this.onReturnClick} would be onClick={this.props.onReturnClick}